rollup.config.js

```diff
@@ -0,0 +1,38 @@
+import swc from '@rollup/plugin-swc';
+
+export default {
+  input: 'src/index.js',
+  output: [{
+    file: 'es/index.js',
+    format: 'es',
+  }, {
+    file: 'lib/index.js',
+    format: 'cjs',
+  }],
+  plugins: [swc({
+    env: {
+      targets: [
+        '>0.2%',
+        'not dead',
+        'not ie <= 8',
+        'not op_mini all',
+      ],
+    },
+  })],
+  external: [
+    'lodash/get',
+    'lodash/reduce',
+    'lodash/split',
+    'lodash/includes',
+    'lodash/endsWith',
+    'lodash/isEqual',
+    'lodash/toString',
+    'lodash/trim',
+    'lodash/findIndex',
+    'lodash/concat',
+    'lodash/join',
+    'lodash/curry',
+    'lodash/cloneDeep',
+    'lodash/set',
+  ],
```

Comment:
could we set 
```
external:['lodash']
```
e?

Reply:
@sangquangnguyen We can use Regex `external: /lodash\/.*/,`, I just finished the update, thanks for you suggestion anh 💯

Reply:
just my curiosity, why can't we just set `external:['lodash']` @truongnguyen-eh  e?

Reply:
@sangquangnguyen Looks like we need to pass the exact imports which are used inside `src/index.js` into `external` config as [mentioned](https://rollupjs.org/configuration-options/#external) in the document. I attempted to apply `external:['lodash']` but received the warning from termial a
![image](https://github.com/user-attachments/assets/812cca9c-284f-4106-b1c9-823e7c897124)

Reply:
Thanks @truongnguyen-eh 🚀